src/app/member/components/MemberDetials.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { Card, CardContent, CardHeader } from "@/components/ui/card";
+import { UserAvatar } from "@/common/Avatar/UserAvatar";
+import { StatusBadge } from "@/common/StatusBadge/StatusBadge";
+import { Briefcase, MapPin, Phone, Shield, User } from "lucide-react";
```

Comment:
use prefex here like Briefcase will be `Briefcase as BriefcaseIcon` because due to this we can not consider 
 it a component in code